### PR TITLE
sogo: 5.11.0 -> 5.11.2

### PR DESCRIPTION
**Pull request by jceb**
Since sogo and the sope library need to be updated together, I combined the changes in on PR. Replaces #347289

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by itepastra — COMMENTED** [review]
Am not in a situation to build and try right now, will look into the results tonight.

Note: can you also remove the `with lib;` in meta since it's an anti-pattern #208242

**Comment by itepastra** [comment]
Result of `nixpkgs-review pr 349490` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>sogo</li>
    <li>sope</li>
  </ul>
</details>

File: pkgs/development/libraries/sope/default.nix [hunk 1]
```diff
@@ -59,6 +51,6 @@ gnustep.stdenv.mkDerivation rec {
     license = licenses.publicDomain;
     homepage = "https://github.com/inverse-inc/sope";
     platforms = platforms.linux;
-    maintainers = [ ];
+    maintainers = with maintainers; [ jceb ];
```
[the remark this answers is not in the document]
Comment (jceb) [reply pkgs/development/libraries/sope/default.nix:54]: Done.

File: pkgs/development/libraries/sope/default.nix [hunk 2]
```diff
@@ -3,23 +3,15 @@
...
-  version = "5.9.0";
+  version = "5.11.2";
 
   src = fetchFromGitHub {
-    owner = "inverse-inc";
+    owner = "Alinto";
     repo = pname;
     rev = "SOPE-${version}";
-    hash = "sha256-JZh8sC/w2MRy3UyWYGMvU47XtWKGnLuUlCsVyyxd7zg=";
+     hash = "sha256-6vec2ZgpK5jcKr3c2SLn6fLAun56MDjupWtR6dMdjag=";
```
[the remark this answers is not in the document]
Comment (jceb) [reply pkgs/development/libraries/sope/default.nix]: Done.

**Review by itepastra — APPROVED** [review]
LGTM

File: pkgs/development/libraries/sope/default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (khaneliman) [reply pkgs/development/libraries/sope/default.nix]: This commit title makes it sound like you are adding yourself to the maintainers list for nixpkgs. You'd want it to be specific to this package.

[the remark this answers is not in the document]
Comment (jceb) [reply pkgs/development/libraries/sope/default.nix]: @khaneliman it was suggested by @itepastra Please make up your mind.

[the remark this answers is not in the document]
Comment (itepastra) [reply pkgs/development/libraries/sope/default.nix]: yeah, I was wrong, @khaneliman's version is better. so `sope: add jceb to maintainers`

**Review by FliegendeWurst — APPROVED** [review]

**Comment by jceb** [comment]
Is there anything else missing before the PR can be merged?

**Review by itepastra — APPROVED** [review]

**Review by misuzu — APPROVED** [review]

**Pull request merged**